knife/Gemfile

```diff
@@ -22,6 +22,6 @@ group(:chefstyle) do
 end
 
 gem "ohai", git: "https://github.com/chef/ohai.git", branch: "18-stable"
-gem "chef", path: ".."
-gem "chef-utils", path: File.expand_path("../chef-utils", __dir__) if File.exist?(File.expand_path("../chef-utils", __dir__))
-gem "chef-config", path: File.expand_path("../chef-config", __dir__) if File.exist?(File.expand_path("../chef-config", __dir__))
+# gem "chef", path: ".."
```

Comment:
let's not keep any commented code. We can always get things from commit history when needed.